sonic_platform_base/sonic_sfp/sfputilbase.py

```diff
@@ -321,7 +321,7 @@ def _read_eeprom_specific_bytes(self, sysfsfile_eeprom, offset, num_bytes):
             sysfsfile_eeprom.seek(offset)
             raw = sysfsfile_eeprom.read(num_bytes)
         except IOError:
-            print("Error: reading sysfs file %s" % sysfs_sfp_i2c_client_eeprom_path)
+            print("Error: reading sysfs file %s")
```

Comment:
Why leave the `%s` formatter in the message with no variable to format?

Is the issue that `sysfs_sfp_i2c_client_eeprom_path` can be `None`? If so, I'd prefer you perform a check for `None` and add the path to the message if the path is not `None`.

Reply:
sysfs_sfp_i2c_client_eeprom_path is undefined.

Reply:
By "undefined" do you mean it is `None`? It should get defined in `_get_port_eeprom_path()`. Can you provide the backtrace from the crash you're seeing?

Reply:
Nov  3 17:36:14.436154 sonic INFO pmon#supervisord: xcvrd   File "/usr/local/lib/python2.7/dist-packages/sonic_sfp/sfputilbase.py", line 323, in _read_eeprom_specific_bytes
Nov  3 17:36:14.436168 sonic INFO pmon#supervisord: xcvrd     print("Error: reading sysfs file %s" % sysfs_sfp_i2c_client_eeprom_path)
Nov  3 17:36:14.436177 sonic INFO pmon#supervisord: xcvrd NameError: global name 'sysfs_sfp_i2c_client_eeprom_path' is not defined
Nov  3 17:36:15.445584 sonic INFO pmon#supervisor-proc-exit-listener: Process xcvrd exited unxepectedly. Terminating supervisor...

Reply:
I see. You've reached the exception case _because_ `sysfs_sfp_i2c_client_eeprom_path` is not defined, so printing it out doesn't make sense.